extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AccessLogConfig.java

```diff
@@ -14,6 +14,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean enabled;
 
+    /**
+     * A regular expression that can be used to exclude some paths from logging.
+     */
+    @ConfigItem
+    Optional<String> excludePattern;
```

Comment:
I created https://github.com/smallrye/smallrye-config/pull/558 which should allow using `Optional<Pattern> excludePattern` here